heat/fft/fft.py

```diff
@@ -0,0 +1,787 @@
+"""Provides a collection of Discrete Fast Fourier Transforms (DFFT) and their inverses."""
+
+import torch
```

Comment:
I saw the hint "Supports torch.half and torch.chalf on CUDA with GPU Architecture SM53 or greater. However it only supports powers of 2 signal length in every transformed dimensions." in the PyTorch docs on fft/fftn ... does this have any implications for us? Or is this such an exotic case that we dont cover anyway...

Reply:
Let's keep this in mind when we tackle #94